shared/api/cluster.go

```diff
@@ -82,12 +82,6 @@ type ClusterPut struct {
 	// API extension: clustering_join
 	ServerAddress string `json:"server_address" yaml:"server_address"`
 
-	// The trust password of the cluster you're trying to join (deprecated, use cluster_token)
-	// Example: blah
-	//
-	// API extension: clustering_join
-	ClusterPassword string `json:"cluster_password" yaml:"cluster_password"` // Deprecated, use ClusterToken.
```

Comment:
@simondeziel so you didnt find any justifiable use of this field from an external client perspective, all former uses were intra-cluster only?

Reply:
> @simondeziel so you didnt find any justifiable use of this field from an external client perspective, all former uses were intra-cluster only?

In our 1:1, it seems you did the search for that field in `stable-5.21` which is why you had many hits. In `main`, that's the only place that appeared.